CONTRIBUTING.md

```diff
@@ -118,13 +117,13 @@ accordingly in the steps below.)
 9. If there are merge conflicts, or a branch lags too much behind V's master,
    you can do the following:
 
-    1. `git pull --rebase origin master` # solve conflicts and do
-       `git rebase --continue`
-    2. `git push pullrequest -f` # this will overwrite your current remote branch
-       with the updated version of your changes.
+   1. `git pull --rebase origin master` # solve conflicts and do
+      `git rebase --continue`
+   2. `git push pullrequest -f` # this will overwrite your current remote branch
+      with the updated version of your changes.
 
 The point of doing the above steps, is to never directly push to the main V
-repository, *only to your own fork*. Since your local `master` branch tracks the
+repository, _only to your own fork_. Since your local `master` branch tracks the
```

Comment:
Why change this?

Reply:
An automatic change that I didn't review properly before submitting, sorry.